### DISCUSSION TITLE
Unipc hires error

### DISCUSSION THREAD

**Discussion by DarqueLilly** [vladmandic/sdnext#2062] [General]
so I use to be able to create highly detailed images taking advantage of  UniPC, highres and dynamic thresh holding but as of late, vladSD keeps giving me this error (RuntimeError: UniPC does not support Hires Fix. Auto WebUI silently swaps to DDIM for this, which DynThresh does not support. Please swap to a sampler capable of img2img processing for HR Fix to work.)  is this an issue your working on or is it just the new way of vladSD?

**Comment by vladmandic** [comment]
Provide actual log, but what tim guessing is that issue is thar dynamic thresholding extension does thar, not SDnext. SDnext supports UniPC for all operations since several months ago.

**Reply by DarqueLilly** [reply]
10:37:36-335722 ERROR    Running script process batch:
                         Y:\01-VladStableDiffusion\automatic\extensions-builtin\sd-dynamic-thresholding\script
                         s\dynamic_thresholding.py: RuntimeError
╭──────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────╮
│ Y:\01-VladStableDiffusion\automatic\modules\scripts.py:451 in process_batch                                │
│                                                                                                            │
│   450 │   │   │   │   args = p.per_script_args.get(script.title(), p.script_args[script.args_f             │
│ ❱ 451 │   │   │   │   script.process_batch(p, *args, **kwargs)                                             │
│   452 │   │   │   │   s.append(f'{script.title()}:{round(time.time()-t0, 2)}s')                            │
│                                                                                                            │
│ Y:\01-VladStableDiffusion\automatic\extensions-builtin\sd-dynamic-thresholding\scripts\dynamic_thresholdin │
│ g.py:101 in process_batch                                                                                  │
│                                                                                                            │
│   100 │   │   if 'UniPC' in (orig_sampler_name, orig_latent_sampler_name) and p.enable_hr:                 │
│ ❱ 101 │   │   │   raise RuntimeError(f"UniPC does not support Hires Fix. Auto WebUI silently s             │
│   102 │   │   mimic_scale = getattr(p, 'dynthres_mimic_scale', mimic_scale)                                │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
RuntimeError: UniPC does not support Hires Fix. Auto WebUI silently swaps to DDIM for this, which DynThresh
does not support. Please swap to a sampler capable of img2img processing for HR Fix to work.
100%|████████████████████████████████████████████████████████████████████████| 31/31 [00:10<00:00,  2.93it/s]
Progress 0.92it/s ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00 0:00:21
10:38:11-342643 INFO
10:38:11-385299 INFO     0: 640x448 1 face, 8.9ms
10:38:11-387284 INFO     Speed: 2.5ms preprocess, 8.9ms inference, 1.5ms postprocess per image at shape (1, 3,
                         640, 448)
100%|████████████████████████████████████████████████████████████████████████| 13/13 [00:04<00:00,  2.90it/s]
10:38:16-199908 INFO     GPU high memory utilization: 96% {'ram': {'used': 3.94, 'total': 127.69}, 'gpu':
                         {'used': 7.67, 'total': 8.0}, 'retries': 2, 'oom': 0}

**Reply by DarqueLilly** [reply]
I thought i fixed it but its still doing the same thing
